tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -482,6 +460,559 @@ There are <?php echo $variable; ?> examples
 
 `;
 
+exports[`mixed-8.php 1`] = `
+<div>
+    <?php
+    echo "1";
+    echo "2";
+    ?>
+</div>
+
+<div>
+    <?php echo "1"; echo "2"; ?>
+</div>
+
+<div>
+  <?php echo "1"; ?>
+  <?php echo "2"; ?>
+</div>
+
+<div>
+    <?php function func($x){
+        return $x;
+    }
+    function other_func($x){
+        return $x;
+    } ?>
+</div>
+
+<div>
+    <?php function func($x){
+        return $x;
+    } ?>
+    <?php function other_func($x){
+        return $x;
+    } ?>
+</div>
+
+<div>
+    <?php echo "1"; ?>
+</div>
+<div>
+    <?php
+    echo "1";
+    ?>
+</div>
+<div>
+    <?php
+    echo "1";
+    echo "2";
+    ?>
+</div>
+<div>
+    <?php
+    echo "1";
+    echo "2";
+    echo "3";
+    ?>
+</div>
+<div>
+    <?php
+    echo "1";
+    echo "2";
+    echo "3";
+    echo "4";
+    echo "5";
+    ?>
+</div>
+<div>
+  <div>
+    <div>
+      <div>
+          <?php echo "1"; ?>
+      </div>
+    </div>
+  </div>
+</div>
+<div>
+  <div>
+    <div>
+      <div>
+          <?php
+          echo "1";
+          echo "2";
+          ?>
+      </div>
+    </div>
+  </div>
+</div>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<div>
+    <?php echo "1";
```

Comment:
Would it be possible to preserve the linebreak after `<?php`?
```php
<?php
echo "1";
echo "2";
?>
```

Reply:
@czosel it is hard, works on this right now, maybe impossible

Reply:
I do it :+1: